packages/engine/src/avatar/functions/moveAvatar.test.tsx

```diff
@@ -87,16 +86,16 @@ describe('moveAvatar function tests', () => {
         parentUUID: getComponent(sceneEntity, UUIDComponent),
         position: new Vector3(),
         rotation: new Quaternion(),
-        entityUUID: Engine.instance.userID as string as EntityUUID,
+        entityUUID: Engine.instance.store.userID as string as EntityUUID,
```

Comment:
i think we need to append `_avatar` to the end of the entityUUID for it to be aligned with how avatars exist at runtime, then we can use `AvatarComponent.getUserAvatarEntity()` correctly